indico/web/client/js/react/components/DatePicker.jsx

```diff
@@ -96,11 +101,15 @@ DatePicker.propTypes = {
   format: PropTypes.string,
   onChange: PropTypes.func.isRequired,
   value: PropTypes.any,
+  min: PropTypes.string,
```

Comment:
It feels a bit strange that "outside the current month" days (which can be selected) and "outside the allowed range" days (which can't be selected) look exactly the same...

![image](https://github.com/indico/indico/assets/179599/1f36f6f2-82d0-445d-9fd3-82d50991fcec)

Reply:
Yeah, that's a fair remark. I'll see what I can do about it.